packages/react/src/components/molecules/CalloutLink/index.js

```diff
@@ -15,8 +15,8 @@ const CalloutLink = (calloutLink) => {
   const classNames = `ma__callout-link${calloutLinkTheme}`;
 
   return(
-    <div className={classNames} title={calloutLink.info}>
-      <a href={calloutLink.href}>
+    <div className={classNames}>
+      <a href={calloutLink.href} title={calloutLink.text}>
```

Comment:
```suggestion
      <a href={calloutLink.href} title={calloutLink.info}>
```

this will be the tooltip text on hover, which lets user to add addtional info

Reply:
https://mayflower.digital.mass.gov/react/index.html?path=/story/molecules-calloutlink--callout-link-example